frontend/src/pages/Employees/AddCourseButton/AddCourseDialog/CoursesList/CoursesList.tsx

```diff
@@ -0,0 +1,61 @@
+import { FC, BaseSyntheticEvent } from 'react';
+
+import { LOADER } from 'constants/loaderTypes';
+import { Course, CoursesListType, ICheckedCourse } from 'types/course';
+import Loader from 'components/Loader';
+
+import { CourseItem } from './CourseItem';
+import { StyledList, StyledHorizontalLine } from './styled';
+
+interface IProps {
+  courses: Course[] | ICheckedCourse[];
+  checkedCourses: ICheckedCourse[];
+  type: CoursesListType;
+  handleCheckboxChange: (event: BaseSyntheticEvent) => void;
+  isCoursesLoading?: boolean;
+  withUpperLine?: boolean;
+  lastCourseRef?: (node?: Element | null) => void;
+}
+
+const CoursesList: FC<IProps> = ({
+  courses,
+  checkedCourses,
+  type,
+  handleCheckboxChange,
+  isCoursesLoading,
+  withUpperLine,
+  lastCourseRef,
+}) => (
+  <>
+    {withUpperLine && !!checkedCourses.length && <StyledHorizontalLine />}
```

Comment:
```suggestion
    {withUpperLine && checkedCourses.length && <StyledHorizontalLine />}
```

Reply:
with this changes we will see 0 in screen. here should be boolean type. operator && return first falsy value. This value will be checkedCourses.length = 0